src/libs/OptionsListUtils.js

```diff
@@ -332,7 +338,14 @@ function getSearchText(report, reportName, personalDetailList, isChatRoomOrPolic
 
             Array.prototype.push.apply(searchTerms, chatRoomSubtitle.split(/[,\s]/));
         } else {
-            searchTerms = searchTerms.concat(report.participants);
+            const participantAccountIDs = report.participantAccountIDs || [];
+            for (let i = 0; i < participantAccountIDs.length; i++) {
+                const accountID = participantAccountIDs[i];
+
+                if (allPersonalDetails[accountID] && allPersonalDetails[accountID].login) {
+                    searchTerms = searchTerms.concat(allPersonalDetails[accountID].login);
```

Comment:
It would be interesting to also allow search by display name, but i think search improvements are coming in Daniel S's proj